src/Mvc/Mvc.Testing/src/Microsoft.AspNetCore.Mvc.Testing.targets

```diff
@@ -46,6 +46,16 @@
     </ItemGroup>
   </Target>
 
+  <Target Name="_GeneratePublishTestManifest" BeforeTargets="Publish" DependsOnTargets="_ResolveMvcTestProjectReferences">
+    <ItemGroup>
+      <_ManifestProjects Include="%(_ContentRootProjectReferences.FusionName)">
```

Comment:
You might want to use a different item name. e.g. `_PublishManfestProject` or something like that. The current itemgroup would have elements added as part of `PrepareResources`